app/models/game.rb

```diff
@@ -2,12 +2,26 @@
 class Game < ActiveRecord::Base
   belongs_to :white_player, class_name: 'User'
   belongs_to :black_player, class_name: 'User'
+  belongs_to :winner, class_name: 'User'
+
   has_many :chess_pieces
 
   validates :name, presence: true
 
   after_create :populate_board!
 
+  def forfeit_by!(player)
+    if player == white_player
+      self.winner = black_player
+    else
+      self.winner = white_player
+    end
+  end
```

Comment:
Currently this method isn't saving. So I don't think the game is being updated.

Since it's a bang `!` method. I'd go with

`update!(winner: black_player)` ...etc...